src/super_gradients/training/transforms/transforms.py

```diff
@@ -618,22 +618,23 @@ def __init__(self, input_dim, swap=(2, 0, 1), max_targets=50):
         self.max_targets = max_targets
 
     def __call__(self, sample):
-        img, target = sample["image"], sample["target"]
-        if len(target) == 0:
-            new_target = np.zeros((self.max_targets, 5), dtype=np.float32)
-        else:
-            new_target = target.copy()
+        img, targets, crowd_targets = sample["image"], sample["target"], sample.get("crowd_target", [])
+        new_targets = np.zeros((self.max_targets, 5), dtype=np.float32) if len(targets) == 0 else targets.copy()
+        new_crowd_targets = np.zeros((self.max_targets, 5), dtype=np.float32) if len(crowd_targets) == 0 else crowd_targets.copy()
 
-        boxes = new_target[:, :4]
-        labels = new_target[:, 4]
         img, r = rescale_and_pad_to_size(img, self.input_dim, self.swap)
-        boxes = xyxy2cxcywh(boxes)
-        boxes *= r
-        boxes = cxcywh2xyxy(boxes)
-        new_target = np.concatenate((boxes, labels[:, np.newaxis]), 1)
+
+        def _pad_rescale_target(targets_in):
```

Comment:
This isn't clean, lets take it out and pass r as an argument.
Also ad docs to it.

Reply:
Changing the function to be a method here d1a317718140d2b6e8ebcd47456823a3fb9a1ee7
Doc added here 15164be538e7291878fc9f0a2367b4e20a0f15d2